src/main/windows/help.ts

```diff
@@ -27,9 +27,16 @@ async function getHelpFileForLang(locale: string) {
 
 let win: BrowserWindow | null = null
 
-export async function openHelpWindow(locale: string) {
+export async function openHelpWindow(locale: string, anchor?: string) {
   if (win) {
     win.focus()
+    if (anchor) {
+      win.webContents.executeJavaScript(`
+        document.getElementById(atob("${btoa(
+          anchor
+        )}"))?.scrollIntoView({"behavior":"smooth"})
```

Comment:
when opening the help window from "learn more", there is a small flickering for a fraction of a second, looks as if things scroll down fast.

this animation is needless and not helping, it would be better, to just open the help at the given position directly, without scrolling for flickering.

maybe `{"behavior":"instant"}`(cmp. https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView ) is the better option here?

Reply:
I thought it would be better if it reacts somehow, but sure we can set this to instant, though maybe users will think nothing changed then?

Reply:
also if the window is not already open it is instant.

Reply:
> also if the window is not already open it is instant

it is not for me (macos, m1 pro), this is the flickering i mentioned

but it is still okayish, and yes, having some feedback when the help is already open is nice.

so, let's leave things as is, it is good enough :)

Reply:
There are two different cases if it is not open it scrolled instantly and if it is already open it scrolled smoothly.
Anyways now I removed the smooth scroll, but I'm not sure now if your issue is not a different one?

Reply:
solved by recent commits